xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -1646,6 +1665,8 @@ bool CActiveAE::InitSink()
     CLog::Log(LOGERROR, "ActiveAE::%s - failed to init", __FUNCTION__);
     m_stats.SetSinkCacheTotal(0);
     m_stats.SetSinkLatency(0);
+    AEAudioFormat invalidFormat;
+    m_stats.SetCurrentSinkFormat(invalidFormat);
```

Comment:
why do you pass by reference? pass by value and you don't need this workaround. You rely on the constructor to init the object as invalid. This can easily break.